tests/TestCase/Http/Client/ResponseTest.php

```diff
@@ -354,13 +354,13 @@ public function testGetCookies()
 
         $result = $response->getCookieData('expiring');
         $this->assertSame('soon', $result['value']);
-        $this->assertTrue($result['httponly']);
-        $this->assertTrue($result['secure']);
+        $this->assertTrue($result['options']['httponly']);
+        $this->assertTrue($result['options']['secure']);
```

Comment:
This options key and renaming expire to expires is going to break applications. Could we keep this interface the same and add in `samesite`?